app/actions/remote/reactions.ts

```diff
@@ -16,6 +16,37 @@ import {forceLogoutIfNecessary} from './session';
 import type {Model} from '@nozbe/watermelondb';
 import type PostModel from '@typings/database/models/servers/post';
 
+export async function getIsReactionAlreadyAddedToPost(serverUrl: string, postId: string, emojiName: string) {
+    try {
+        const {database} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
+
+        const currentUserId = await getCurrentUserId(database);
+        const emojiAlias = getEmojiFirstAlias(emojiName);
+        const isReactionAlreadyAddedToPost = await queryReaction(database, emojiAlias, postId, currentUserId).fetchCount() > 0;
+
+        return isReactionAlreadyAddedToPost;
```

Comment:
```suggestion
        return queryReaction(database, emojiAlias, postId, currentUserId).fetchCount() > 0;
```

Reply:
Since the return value of `fetchCount` is a `Promise<number>` and it needs to be compared with 0, I did not remove await.